smoke-tests/SmokeTestGenerator.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * This script auto-generates bash/cmd scripts meant to test smoke test the entire plugin. These scripts will do so by
+ * running a series of operations that collectively capture a vertical slice of the plugin, hitting every major piece
```

Comment:
Can you add an example on how to invoke?

Reply:
Will do.